common/map_file/nodes/lanelet2_map_loader/lanelet2_map_visualization.cpp

```diff
@@ -53,37 +53,46 @@ void binMapCallback(autoware_lanelet2_msgs::MapBin msg)
   lanelet::ConstLanelets road_lanelets = lanelet::utils::query::roadLanelets(all_lanelets);
   lanelet::ConstLanelets crosswalk_lanelets = lanelet::utils::query::crosswalkLanelets(all_lanelets);
-  std::vector<lanelet::ConstLineString3d> stop_lines = lanelet::utils::query::stopLinesLanelets(road_lanelets);
-  // std::vector<lanelet::TrafficLightConstPtr> tl_reg_elems = lanelet::utils::query::trafficLights(all_lanelets);
-  // std::vector<lanelet::AutowareTrafficLightConstPtr> aw_tl_reg_elems =
-  //     lanelet::utils::query::autowareTrafficLights(all_lanelets);
 
-  std_msgs::ColorRGBA cl_road, cl_cross, cl_ll_borders, cl_stoplines, cl_trafficlights;
+  std::vector<lanelet::ConstLineString3d> tl_stop_lines = lanelet::utils::query::getTrafficLightStopLines(road_lanelets);
+  std::vector<lanelet::ConstLineString3d> ss_stop_lines = lanelet::utils::query::getStopSignStopLines(road_lanelets);
+  std::vector<lanelet::TrafficLightConstPtr> tl_reg_elems = lanelet::utils::query::trafficLights(all_lanelets);
+  std::vector<lanelet::AutowareTrafficLightConstPtr> aw_tl_reg_elems =
+      lanelet::utils::query::autowareTrafficLights(all_lanelets);
+
+  std_msgs::ColorRGBA cl_road, cl_cross, cl_ll_borders, cl_tl_stoplines, cl_ss_stoplines, cl_trafficlights;
   setColor(&cl_road, 0.2, 0.7, 0.7, 0.3);
   setColor(&cl_cross, 0.2, 0.7, 0.2, 0.3);
   setColor(&cl_ll_borders, 1.0, 1.0, 1.0, 1.0);
-  setColor(&cl_stoplines, 1.0, 0.0, 0.0, 0.5);
+  setColor(&cl_tl_stoplines, 1.0, 0.5, 0.0, 0.5);
+  setColor(&cl_ss_stoplines, 1.0, 0.0, 0.0, 0.5);
   setColor(&cl_trafficlights, 0.7, 0.7, 0.7, 0.8);
 
   visualization_msgs::MarkerArray map_marker_array;
 
   insertMarkerArray(&map_marker_array, lanelet::visualization::laneletsBoundaryAsMarkerArray(
-                                           road_lanelets, cl_ll_borders, g_viz_lanelets_centerline));
-  // insertMarkerArray(&map_marker_array, lanelet::visualization::laneletsAsTriangleMarkerArray(
-  //                                          "road_lanelets", road_lanelets, cl_road));
+    road_lanelets, cl_ll_borders, g_viz_lanelets_centerline));
+  insertMarkerArray(&map_marker_array, lanelet::visualization::laneletsAsTriangleMarkerArray(
+    "road_lanelets", road_lanelets, cl_road));
   insertMarkerArray(&map_marker_array, lanelet::visualization::laneletsAsTriangleMarkerArray(
-                                           "crosswalk_lanelets", crosswalk_lanelets, cl_cross));
-  insertMarkerArray(&map_marker_array, lanelet::visualization::laneletDirectionAsMarkerArray(road_lanelets));
-  insertMarkerArray(&map_marker_array, lanelet::visualization::lineStringsAsMarkerArray(stop_lines, "stop_lines",
-                                                                                              cl_stoplines, 0.5));
-  // insertMarkerArray(&map_marker_array, lanelet::visualization::autowareTrafficLightsAsMarkerArray(
-  //                                          aw_tl_reg_elems, cl_trafficlights));
```

Comment:
No need to change, but I am writing a note for reference. We commented these lines out because they would cause a segfault when a map did not contain the required information. I'm hoping the new version of autoware makes them fail more gracefully. We will test it.